lua/randomat2/sh_events/sh_poker.lua

```diff
@@ -0,0 +1,258 @@
+-- Logan Christianson
+
+ConVars = ConVars or {}
```

Comment:
I'm concerned about such a generic global. Any addon could use the same name and overwrite your stuff.

Consider putting all these globals (tables, properties, methods) in a namespace such as `POKER`

Reply:
I'll rename the global so it's a bit less generic, but even a namespace name like that could still run into issues with someone else trying to write their own poker game

Reply:
Then call it something like `RandomatPoker` or `LoganPoker`. I'm more concerned about the global pollution with all the methods and properties.